consensus/src/threshold_simplex/mocks/application.rs

```diff
@@ -197,7 +197,7 @@ impl<E: Clock + RngCore + Spawner, H: Hasher, P: PublicKey> Application<E, H, P>
             .await;
 
         // Generate the payload
-        let rand = self.context.gen::<u64>();
+        let rand = self.context.r#gen::<u64>();
```

Comment:
Should be done in a follow-up, but we're several minor versions behind on `rand` and should upgrade (and quite a few other deps...)

They deprecated `gen` in `0.9.0` in preparation for the new edition, which'd get rid of the ugly raw `r#` syntax all over the place: https://docs.rs/rand/latest/rand/trait.Rng.html#method.gen